<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditListWidget</class>
 <widget class="QWidget" name="EditListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add" resource="../resources/resources.qrc">
         <normaloff>:/kde_icons/list-add.png</normaloff>:/kde_icons/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove" resource="../resources/resources.qrc">
         <normaloff>:/kde_icons/list-remove.png</normaloff>:/kde_icons/list-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveUpPushButton">
       <property name="text">
        <string>Move &amp;Up</string>
       </property>
       <property name="icon">
        <iconset theme="arrow-up" resource="../resources/resources.qrc">
         <normaloff>:/kde_icons/arrow-up.png</normaloff>:/kde_icons/arrow-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveDownPushButton">
       <property name="text">
        <string>Move &amp;Down</string>
       </property>
       <property name="icon">
        <iconset theme="arrow-down" resource="../resources/resources.qrc">
         <normaloff>:/kde_icons/arrow-down.png</normaloff>:/kde_icons/arrow-down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>EditListWidget</receiver>
   <slot>slotCurrentRowChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditListWidget</receiver>
   <slot>slotAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removePushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditListWidget</receiver>
   <slot>slotRemove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveUpPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditListWidget</receiver>
   <slot>slotMoveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveDownPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditListWidget</receiver>
   <slot>slotMoveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCurrentRowChanged()</slot>
  <slot>slotAdd()</slot>
  <slot>slotRemove()</slot>
  <slot>slotMoveUp()</slot>
  <slot>slotMoveDown()</slot>
 </slots>
</ui>
